<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDialog</class>
 <widget class="QDialog" name="KeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>按键映射</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/ico/ico.ico"/>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>上一首</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="key_stop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>继续</string>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="key_next">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>1</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="key_prev">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="key_play">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>139</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清除所有</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
